lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -69,14 +72,17 @@ def test_user_status(self):
         # test for correct status when no error returned
         user = UserFactory.create()
         status = IDVerificationService.user_status(user)
-        expected_status = {'status': 'none', 'error': '', 'should_display': True, 'verification_expiry': ''}
+        expected_status = {'status': 'none', 'error': '', 'should_display': True, 'verification_expiry': '',
+                           'status_date': ''}
         self.assertDictEqual(status, expected_status)
 
-        # test for when photo verification has been created
-        SoftwareSecurePhotoVerification.objects.create(user=user, status='approved')
-        status = IDVerificationService.user_status(user)
-        expected_status = {'status': 'approved', 'error': '', 'should_display': True, 'verification_expiry': ''}
-        self.assertDictEqual(status, expected_status)
+        with freeze_time('2015-01-02'):
```

Comment:
Can you explain why this was necessary? Seeing that mixed with the datetime.now(utc) seems odd, but is that just because datetime.now(utc) is returning the frozen time? Maybe a comment to specify that?